plugins/scaffolder-react/alpha-api-report.md

```diff
@@ -288,6 +288,19 @@ export interface TemplateGroupsProps {
   templateFilter?: (entity: TemplateEntityV1beta3) => boolean;
 }
 
+// @alpha
+export const useFeatureFlaggedProperties: (
```

Comment:
`useFilteredSchemaProperties` or something? Maybe don't have to mention feature flags in the hook itself actually. Thinking it's a little cleaner as to what it's doing.